src/main/java/org/mule/extension/objectstore/api/ExtensionObjectStore.java

```diff
@@ -156,6 +156,12 @@ public void start() throws MuleException {
     storeManagerProvider = getObjectStoreManagerProvider();
     objectStoreManager = getObjectStoreManager();
 
+    if (maxEntries < 0) {
+      LOGGER.warn("The maxEntries parameter should not be negative (given value was: " + maxEntries.toString()
```

Comment:
I'd suggest not concatenating strings, use a format string and use the maxEntries as another parameter of warn